checker/src/main/java/org/checkerframework/checker/initialization/InitializationStore.java

```diff
@@ -48,8 +48,8 @@
      */
     public InitializationStore(CFAbstractAnalysis<V, S, ?> analysis, boolean sequentialSemantics) {
         super(analysis, sequentialSemantics);
-        initializedFields = new HashSet<>(4);
-        invariantFields = new HashMap<>(4);
+        initializedFields = new HashSet<>(5);
```

Comment:
Nice catch! But I don't think this `4` is from the number of qualifiers. This seems to be an arbitrary value for how many fields are expected. Maybe add a comment instead.

Reply:
already add a comment :)